### PR TITLE
Update, from msgpack-lite to  notepack.io (support new version of soc…

### PR DESCRIPTION
**Pull request by v4l3r10**
…keti.io-redis)

Fix, removed old namespace tag
Update dependencies

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -137,12 +137,12 @@ Emitter.prototype.emit = function(){
...
   };
   var chn = this.prefix + '#' + packet.nsp + '#';
-  var msg = msgpack.encode([uid, packet, opts]);
+  var msg = notepack.encode([uid, packet, opts]);
 
   // publish
   if (opts.rooms && opts.rooms.length) {
     opts.rooms.forEach(function(room) {
-      var chnRoom = chn + room + '#';
+      var chnRoom = chn + room;
```
Comment (darrachequesne) [inline index.js]: Could you please update this part, to match https://github.com/socketio/socket.io-redis/blob/5.0.0/index.js#L407 (related: https://github.com/socketio/socket.io-redis/pull/151)?

File: index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
 var client = require('redis').createClient;
 var parser = require('socket.io-parser');
 var hasBin = require('has-binary');
-var msgpack = require('msgpack-lite');
+var notepack = require('notepack.io');
```
Comment (darrachequesne) [inline index.js]: Since notepack is just another msgpack implementation, how about keeping the `var msgpack` here?

Comment (v4l3r10) [inline index.js]: It looks like a wise choice! like socket.io-redis (https://github.com/socketio/socket.io-redis/blob/master/index.js#L8)

File: index.js (hunk 1 above)
Comment (v4l3r10) [inline index.js]: Fixed

**Comment by darrachequesne** [comment]
@v4l3r10 great, thanks!

**Pull request merged**